test/ValidateID.cpp

```diff
@@ -821,6 +821,53 @@ TEST_F(ValidateID, OpStoreTypeBad) {
   CHECK(spirv, SPV_ERROR_INVALID_ID);
 }
 
+TEST_F(ValidateID, OpStoreVoid) {
```

Comment:
This should follow the naming convention of other Tests by adding the "Bad" Suffix